CommonTools/PileupAlgos/interface/PuppiCandidate.h

```diff
@@ -0,0 +1,23 @@
+#ifndef CommonTools_PileupAlgos_PuppiCandidate
+#define CommonTools_PileupAlgos_PuppiCandidate
+
+#include "fastjet/PseudoJet.hh"
+
+const double pseudojet_invalid_eta = -1e200;
```

Comment:
can this code use the already available pseudojet_invalid_rap?
If the available constant can not be reused, for consistency, better keep it in the same namespace